hub-detect/src/main/resources/hub-detect-4-sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
```

Comment:
Any consideration to using "set -e" and "set -u"?

-e      Exit immediately if a simple command (see SHELL GRAMMAR above) exits with a non-zero status.  The shell does not exit if the command that fails is part of the command list immediately following a while or until keyword, part of the test in an if statement, part of a && or || list, or  if  the  command's  return value is being inverted via !.  A trap on ERR, if set, is executed before the shell exits.
-u      Treat unset variables as an error when performing parameter expansion.  If expansion is attempted on an unset variable, the shell prints an error message, and, if not interactive, exits with a non-zero status.

Reply:
Since we're going to validate DETECT_SOURCE, we'll leave this alone for now. If we can find some more value here, it's a good idea for the future.